common.mk

```diff
@@ -6,7 +6,7 @@ SHELL=/bin/bash
 #########################################################################################
 # variables to get all *.scala files
 #########################################################################################
-lookup_scala_srcs = $(shell find -L $(1)/ -name target -prune -o -iname "*.scala" -print 2> /dev/null)
+lookup_scala_srcs = $(shell find -L $(1)/ -name target -prune -o \( -iname "*.scala" -o -iname "*.cc" -o -iname "*.v" \) -print 2> /dev/null)
```

Comment:
IMO this makes this cluttered (one variable `lookup_scala_srcs` does everything). I think it would be better to split it up into multiple variables. I currently do that for the other work that I'm doing.

Reply:
I don't see why that's necessary, since there appears to be no place where you care about one set of files, but not the other.
I will rename this function to `lookup_sbt_deps`, which is more accurate, as this function returns files which should trigger SBT re-runs when modified.